### PR TITLE
Add CC13X2_26X2 to github C/I

### PR DESCRIPTION
**Pull request by srickardti**
#### Problem

The Texas Instruments CC13X2_26X2 platform is not currently part of the automated build system within github workflows

#### Change overview

* Added the TI SimpleLink CC13XX CC26XX SDK as a git submodule
* Added a github workflow to build the examples and report the bloat change
* Created a docker image with SysConfig installed for building

#### Testing

The github workflow was tested on a personal repository. That commit is intact on this branch, but it downloads and installs SysConfig each time.

The docker image was tested manually to ensure SysConfig installed correctly. The image is not currently published on dockerhub so the build will fail the first time until someone with CSA organization credentials pushes the image.

### CODE REVIEW

**Review by tlykkeberg-grundfos — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni [hunk 1]
```diff
@@ -380,18 +380,18 @@ template("ti_simplelink_sdk") {
 
     if (ti_simplelink_device_family == "cc13x2x7_26x2x7") {
       libs += [
-        "${ti_simplelink_sdk_root}/source/ti/dmm/lib/gcc/m4f/libdmmlib_freertos.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/OneLib.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/StackWrapper.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/ble_r2.symbols",
+        rebase_path(ti_simplelink_sdk_root + "/source/ti/dmm/lib/gcc/m4f/libdmmlib_freertos.a"),
```
Comment (mspang) [inline third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni:383]: Why was this changed?

Please don't, this will use absolute paths which isn't desirable

"args" should be rebased to root_build_dir in the build dir

"libs" will be rebased to root_build_dir by GN so it should not be rebased in the build file

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni [hunk 2]
```diff
@@ -81,7 +81,7 @@ template("ti_sysconfig") {
     args = [
       ti_sysconfig_root,
       "-s",
-      ti_simplelink_sdk_root + "/.metadata/product.json",
+      rebase_path(ti_simplelink_sdk_root + "/.metadata/product.json"),
```
Comment (mspang) [inline third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni:84]: rebase_path(..., root_build_dir)

File: third_party/ti_simplelink_sdk/ti_simplelink_executable.gni [hunk 3]
```diff
@@ -52,7 +52,7 @@ template("ti_simplelink_executable") {
       inputs = [ ldscript ]
 
       ldflags = [
-        "-L${ti_simplelink_sdk_root}/source",
+        "-L" + rebase_path(ti_simplelink_sdk_root + "/source"),
```
Comment (mspang) [inline third_party/ti_simplelink_sdk/ti_simplelink_executable.gni:55]: rebase_path(..., root_build_dir)

**Review by mspang — APPROVED** [review]

File: .github/workflows/examples-cc13x2x7_26x2x7.yaml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                  attempt_delay: 2000
+            - name: Set up environment for size reports
+              if: ${{ !env.ACT }}
+              env:
+                  GH_CONTEXT: ${{ toJson(github) }}
+              run: scripts/tools/memory/gh_sizes_environment.py "${GH_CONTEXT}"
+            - name: Install SysConfig
+              timeout-minutes: 30
+              run: |
+                  wget https://software-dl.ti.com/ccs/esd/sysconfig/sysconfig-1.11.0_2225-setup.run
```
Comment (andy31415) [inline .github/workflows/examples-cc13x2x7_26x2x7.yaml:59]: We should have a separate build image for operations like this rather than running it on every build
